src/core/composables/infosModal.js

```diff
@@ -0,0 +1,53 @@
+import compact from 'lodash/compact';
+import get from 'lodash/get';
+import { computed } from 'vue';
+import { E_LEARNING, SHORT_DURATION_H_MM, DD_MM_YYYY, REMOTE } from '@data/constants';
+import { getISOTotalDuration, ascendingSortBy } from '@helpers/dates/utils';
+import CompaniDuration from '@helpers/dates/companiDurations';
+import CompaniDate from '@helpers/dates/companiDates';
+
+export const useInfosModal = (course, slots) => {
+  // make sure code is similar to back part in TrainingContracts helper
```

Comment:
ce n'est plus dans TrainingContracts helper ; je epnse que tu peux juste laisser `// make sure code is similar to back part`

Reply:
done